common.gypi

```diff
@@ -471,6 +471,13 @@
             'ldflags': [
               '-Wl,-blibpath:/usr/lib:/lib:/opt/freeware/lib/pthread/ppc64',
             ],
+            'conditions': [
+              [ 'node_shared=="true"', {
+                'ldflags': [
+                  '-Wl,-bnoipath', # Do not add the full path to libnode.a in executables, trust that the LIBPATH is set correctly by the user at runtime
```

Comment:
I guess this may why I see it fail earlier in the AIX tests, I'll try setting the LIBPATH to see if that helps.

Reply:
@lux01, setting the LIBPATH to $WORKSPACE/out/Release allowed the tests to run, but all of the addon tests failed See https://ci.nodejs.org/job/node-test-commit-aix-shared-lib/1425/ for the output.

Reply:
Looks like the failing tests have errors like:


```shell
ppc64/test/addons/07_factory_of_wrapped_objects/build/Release/addon.node because:
    	Symbol _ZN2v811HandleScope12CreateHandleEPNS_8internal7IsolateEm (number 5) is not exported from dependent
    	  module node.
    	Symbol _ZN2v811HandleScopeC1EPNS_7IsolateE (number 6) is not exported from dependent
    	  module node.
    	Symbol _ZN2v811HandleScopeD1Ev (number 7) is not exported from dependent
    	  module node.
    	Symbol _ZN2v812api_internal12ToLocalEmptyEv (number 8) is not exported from dependent
    	  module node.
    	Symbol _ZN2v812api_internal13DisposeGlobalEPm (number 9) is not exported from dependent
    	  module node.
    	Symbol _ZN2v812api_internal17FromJustIsNothingEv (number 10) is not exported from dependent
    	  module node.
    Additional errors occurred but are not reported.
```